tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -86,27 +86,50 @@ steps:
       echo VERSION_PATCH=$VERSION_PATCH
       echo VERSION_INCLUDE_INTERNAL_VERSIONS=$VERSION_INCLUDE_INTERNAL_VERSIONS
 
-      # Generate the build version
+      # Generate the build version. Sets the environment variables version, codeVersion, and isLatest.
+      # These are referenced in following steps prefixed by this task name. E.g. SetVersion.version
       flub generate buildVersion
 
-- task: Bash@3
-  displayName: Update Package Version
-  env:
-    VERSION_RELEASE: $(release)
-  inputs:
-    targetType: 'inline'
-    workingDirectory: ${{ parameters.buildDirectory }}
-    script: |
-      echo SETVERSION_VERSION=$SETVERSION_VERSION
-      echo SETVERSION_CODEVERSION=$SETVERSION_CODEVERSION
+# Don't use "flub bump" for gitrest, historian, and routerlicious temporarily
+# TODO: AB#2415
```

Comment:
Hmmmm, I think it might be better to special case the client build, which is the only one using a version scheme that requires `flub`.